app/src/main/java/org/fossasia/openevent/app/contract/model/UtilModel.java

```diff
@@ -2,10 +2,20 @@
 
 public interface UtilModel {
```

Comment:
@iamareebjamal No need to make interface UtilModel.. Just define static methods in AndroidUtilModel class. And prefer `Util.java` for class name.